plugins/colorsource/plugin.json

```
@@ -0,0 +1,70 @@
+{
+  "$schema": "../../schemas/plugin.json",
+
+  "name": "ColorSource",
+  "description": [
+    "<p>",
+    "The ColorSource (CS) is a series of DMX consoles by ETC.",
+    "The installed fixture library is not available in a readable format,",
+    "but custom fixtures created using the Personality Editor are saved as JSON files.",
+    "</p>",
+    "<p>",
+    "There is a ColorSource Offline program available that simulates the console if you don't have physical access to one.",
+    "</p>"
+  ],
+  "links": {
+    "Console product page": "https://www.etcconnect.com/Products/Consoles/ColorSource/Features.aspx",
+    "Downloads (including Personality Editor)": "https://www.etcconnect.com/Products/Consoles/ColorSource/Software.aspx",
+    "Console Manual": "https://www.etcconnect.com/WorkArea/DownloadAsset.aspx?id=10737496502",
+    "Personality Editor manual": "https://www.etcconnect.com/WorkArea/DownloadAsset.aspx?id=10737490177"
+  },
+  "fixtureUsage": [
+    "<p>",
+    "Load the <code>userlib.jlib</code> onto the root of a USB drive and plug it into the console",
+    "(or your computer, if you use ColorSource Offline).",
```

```suggestion
    "(or into your computer, if you use ColorSource Offline).",
```

To avoid confusion if I should load the file onto my computer (where?) or plug the USB drive into my computer.